### ISSUE TITLE
the symbol '\' before a header should not display

### ISSUE THREAD

**Issue by bibaodi** [arXiv/html_feedback#5052]
### Description

the symbol '\' here in '\Description' before the  header 'Description' should not be presented. 
in fact, I think the paragraph is duplicate with figure1's description which already displayed.

### (Optional:) Please add any files, screenshots, or other information here.

this screenshot image show origin display result.
<img width="948" height="725" alt="Image" src="https://github.com/user-attachments/assets/15543450-dcdd-47af-915d-99924ce9e5fc" />

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

842ed601-9bd6-40d8-ba6c-f35bd3efa4e9

### Paper URL

https://arxiv.org/html/2406.05068v1

### Browser

Firefox/140.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from brucemiller/LaTeXML#2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
